tools/jxlinfo.c

```diff
@@ -13,7 +13,7 @@
 
 #include "jxl/decode.h"
 
-int PrintBasicInfo(FILE* file) {
+int PrintBasicInfo(FILE* file, int verbose) {
   uint8_t* data = NULL;
   size_t data_size = 0;
   // In how large chunks to read from the file and try decoding the basic info.
```

Comment:
If we're going to install this as an utility, I would perhaps have somewhat bigger chunks than 64 bytes :)

Reply:
Bumped it up to 2 kb.